libs/generic-view/ui/src/lib/interactive/modal/helpers/modal-title-icon.tsx

```diff
@@ -14,6 +14,7 @@ export const ModalTitleIcon: APIFC<undefined, ModalTitleIconConfig> = ({
   children,
   ...rest
 }) => {
+  // @ts-ignore
```

Comment:
Why was this @ts-ignore added? I removed it locally, and TypeScript doesn't throw any errors 🤷‍♂️

Reply:
It doesn't throw until... it throws. Sometimes during hot-reload you can get TS error from here in the console, while in IDE everything works fine 🤷‍♂️ . I tried to fix it but couldn't find the real issue, so I added the ignore command to not be forced to restart the app every time it happens.